velox/core/QueryConfig.h

```diff
@@ -674,6 +677,16 @@ class QueryConfig {
     return value;
   }
 
+  int32_t sparkPartitionId() const {
+    auto id = get<int32_t>(kSparkPartitionId);
+    if (id.has_value()) {
```

Comment:
use VELOX_CHECK(id.has_value(), "Spark partition id is not set.")

Reply:
Updated, thanks.